packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -124,6 +125,9 @@ class NavigationDrawer extends StatelessWidget {
   /// Upon updating [selectedIndex], the [NavigationDrawer] will be rebuilt.
   final ValueChanged<int>? onDestinationSelected;
 
+  /// padding of drawer items
```

Comment:
This doc does not meet our style guide, please read: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc

Reply:
my bad, I'll fix it. I was keep jumping between 2-3 PRs

Reply:
I've updated the doc comments. Please have a look.